vars/Debian.yml

```diff
@@ -1,5 +1,6 @@
 ---
-sensu_apt_package: sensu
+sensu_apt_package:
```

Comment:
Why not plural name instead because it's list.
Or make list on `"{{ sensu_apt_dependencies + [ sensu_apt_package ] }}"`
```suggestion
sensu_apt_packages:
```

Reply:
This ```sensu_apt_package``` won't be a list because it will consist only one package

Reply:
```suggestion
sensu_apt_package: sensu
```